### PR TITLE
instanciate the siteconfig in the test_driver object

### PR DESCRIPTION
**Pull request by dothebart**

### CODE REVIEW

File: release_tester/arangodb/starter/deployments/dc2dc.py [hunk 1]
```diff
@@ -385,8 +387,8 @@ def _mitigate_known_issues(self, last_sync_output):
                 )
             else:
                 self.progress(True, "arangosync: restarting instances...")
-                self.cluster1["instance"].kill_sync_processes()
-                self.cluster2["instance"].kill_sync_processes()
+                self.cluster1["instance"].kill_sync_processes(True, SYNC_VERSIONS["140"])
```
Comment (informalict) [inline release_tester/arangodb/starter/deployments/dc2dc.py:390]: you compare `SYNC_VERSIONS["140"]` to Starter version inside `kill_sync_process` ?

File: release_tester/arangodb/starter/manager.py [hunk 2]
```diff
@@ -659,10 +660,13 @@ def replace_binary_setup_for_upgrade(self, new_install_cfg):
             ]
 
     @step
-    def kill_sync_processes(self):
+    def kill_sync_processes(self, force, rev):
         """kill all arangosync instances we posses"""
         for i in self.all_instances:
             if i.is_sync_instance():
+                if not force and i.pid_file is not None and rev > semver.VersionInfo.parse("0.15.0"):
```
Comment (informalict) [inline release_tester/arangodb/starter/manager.py:667]: should not it be `>=` instead `>`

**Review by informalict — COMMENTED** [review]
LGTM except two little comments

File: release_tester/arangodb/starter/deployments/dc2dc.py (hunk 1 above)
Comment (dothebart) [reply release_tester/arangodb/starter/deployments/dc2dc.py:390]: yes, I probably should change this.

**Review by informalict — APPROVED** [review]
lgtm

**Pull request merged**